### PR TITLE
[SMP-1220] SMP monitoring

### PR DESCRIPTION
**Pull request by brian-f-harness**
Add new SMP monitoring content

For reviewers, [a preview](https://644c3242847cab0eebed188b--harness-developer.netlify.app/docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem) is available.

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by pratmit — COMMENTED** [review]
Made some couple of custom changes.

**Review by pratmit — DISMISSED** [review]
LGTM with the changes proposed.

File: docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem.md [hunk 1]
```diff
@@ -1,142 +1,267 @@
...
+description: Monitor the infrastructure of your installation
 # sidebar_position: 2
 helpdocs_topic_id: ho0c1at9nv
 helpdocs_category_id: 75ydek1suj
 helpdocs_is_private: false
 helpdocs_is_published: true
 ---
 
-You can monitor your Harness Self-Managed Enterprise Edition installation and receive alerts on metrics such as CPU, memory, and disk usage.
+Monitor the infrastructure components of your Harness Self-Managed Enterprise Edition installation by bringing your own open-source monitoring system, such as Prometheus, and integrate with observability tools, such as Grafana.
```
Comment (aimurphy) [inline docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem.md:11]:
```suggestion
Monitor the infrastructure components of your Harness Self-Managed Enterprise Edition installation by bringing your own open-source monitoring system, such as Prometheus, and integrating with observability tools, such as Grafana.
```

File: docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem.md [hunk 2]
```diff
@@ -1,142 +1,267 @@
...
 
-The Harness Self-Managed Enterprise Edition monitoring options available depend on whether you are running Harness Self-Managed Enterprise Edition - Virtual Machine or Harness Self-Managed Enterprise Edition - Kubernetes Cluster.
+To monitor database applications like MongoDB, Postgres, or Redis for the Harness Self-Managed Enterprise Edition, this topic describes how you can use a Prometheus server installed in a Kubernetes cluster outside of Harness services. In this example, the monitored target application is present in one cluster, and Prometheus and Grafana are installed in another cluster.
```
Comment (aimurphy) [inline docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem.md:13]:
```suggestion
To demonstrate how you can monitor database applications, like MongoDB, Postgres, or Redis, for the Harness Self-Managed Enterprise Edition, this topic describes how you can use a Prometheus server installed in a Kubernetes cluster outside of Harness services. In this example, the monitored target application is present in one cluster, and Prometheus and Grafana are installed in another cluster.
```

Optional suggestion  to make it clear that this is an example of one setup and they don't necessarily have to use Prometheus & Grafana.

File: docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem.md [hunk 3]
```diff
@@ -1,142 +1,267 @@
...
+    targetPort:http
+  - name:https
+    nodePort:32505
+    port:443
+    protocol:TCP
+    targetPort:https
 ```
-To view these addresses, log into the VM running Harness, and then view the Kubernetes services running in the `monitoring` namespace:
 
+  :::info note
```
Comment (aimurphy) [inline docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem.md:62]:
```suggestion
  :::info
```

There is a spacing issue here - In the preview this note is broken.

Also, some optional food for thought:

We haven't really decided anything about this in our style guide, but I think the default "info" title is fine and will save you from typing "note" every time. I was taught that "note" has no importance over regular text, so it should be avoided over other more valuable names like Info, Caution, Warning, Tip, etc.

To me, "Note/Info" are like: Here's some interesting information. If I need to do something in order for something else to work, I think it should be formatted as a step or maybe a Caution.

File: docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem.md [hunk 4]
```diff
@@ -1,142 +1,267 @@
...
+          service:
+            name: mongodb-replicaset-chart-metrics
+            port:
+              number: 9216
 ```
-### Prometheus
 
-The Prometheus port number is taken from the `prometheus-k8s` service (in this example, `30900`).
+:::info note
+ Add your IPs to your allow list so the metrics exposed by the ingress are only accessible internally. The IP included in the allow list is the xxternal IP for the node where you host Prometheus in a separate cluster.
```
Comment (aimurphy) [inline docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem.md:98]:
```suggestion
 Add your IPs to your allow list so the metrics exposed by the ingress are only accessible internally. The IP included in the allow list is the external IP for the node where you host Prometheus in a separate cluster.
```

**Review by aimurphy — COMMENTED** [review]
Some items to fix and some optional comments.

File: docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem.md (hunk 3 above)
Comment (KimberlyFields) [reply docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem.md]: The :::info note syntax give you a blue background with "Note" as the heading. The :::note syntax defaults to a white background, which does not stand out. Using :::info note is a hack to make a note stand out. https://harness.atlassian.net/wiki/spaces/DOCS/pages/21327872338/Tips+hints+and+best+practices#Notes

Comment (aimurphy) [reply docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem.md]: But do we prefer "Note" as the label over "Info"? Or is it up to the author?

**Review by aimurphy — DISMISSED** [review]

**Pull request merged**